### PR TITLE
Edit Hosting type Create View

### PR DESCRIPTION
**Pull request by RahulJha-mastek**
Story AB#15299
Task AB#15857

### CODE REVIEW

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Controllers/CatalogueSolutionsController.cs [hunk 1]
```diff
@@ -160,5 +161,39 @@ public async Task<IActionResult> Roadmap(CatalogueItemId solutionId, RoadmapMode
 
             return RedirectToAction(nameof(ManageCatalogueSolution), new { solutionId });
         }
+
+        [HttpGet("manage/{solutionId}/hosting-type")]
+        public async Task<IActionResult> HostingType(CatalogueItemId solutionId)
+        {
+            var solution = await solutionsService.GetSolution(solutionId);
+
+            if (solution == null)
```
Comment (bjss-chris) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Controllers/CatalogueSolutionsController.cs:170]: Prefer `solution is null` here.

File: tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Areas/Admin/Models/HostingTypeSectionModelTests.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+            var actual = model.StatusHostingType();
+
+            actual.Should().Be(FeatureCompletionStatus.Completed);
+        }
+
+        [Fact]
+        public static void StatusHostingType_NoCloudTypeAdded_ReturnsNotStarted()
+        {
+            var model = new HostingTypeSectionModel { };
```
Comment (bjss-chris) [inline tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Areas/Admin/Models/HostingTypeSectionModelTests.cs:61]: Object initializer is redundant here.

**Review by bjss-chris — CHANGES_REQUESTED** [review]
Only two small things.

File: tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Areas/Admin/Models/HostingTypeSectionModelTests.cs (hunk 2 above)
Comment (RahulJha-mastek) [reply tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Areas/Admin/Models/HostingTypeSectionModelTests.cs]: Used `new HostingTypeSectionModel()`

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Controllers/CatalogueSolutionsController.cs (hunk 1 above)
Comment (RahulJha-mastek) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Controllers/CatalogueSolutionsController.cs]: Updated to `solution is null`

**Review by bjss-chris — APPROVED** [review]

**Comment by bjss-chris** [comment]
Please fix spelling in commit message before merge

**Pull request merged**